src/client/dfuse/dfuse_main.c

```diff
@@ -251,70 +251,104 @@ static void
-	    "thread for a single event queue for a total of two threads\n"
+	    "Threading and resource usage:\n"
+	    "dfuse has two types of threads: fuse threads which accept and process requests from\n"
+	    "the kernel, and progress threads which complete asynchronous read/write operations.\n"
+	    "Each asynchronous progress thread uses one DAOS event queue to consume additional\n"
+	    "network resources. As all metadata operations are blocking, the level of concurrency\n"
+	    "in dfuse is limited by the number of fuse threads.\n"
+	    "By default, the total thread count is one per available core to allow maximum\n"
+	    "throughput. If hyperthreading is enabled, then one thread per hyperthread core\n"
+	    "is used. This can be modified in two ways: Reducing the number of available\n"
+	    "cores by running dfuse in a cpuset via numactl or similar tools,\n"
+	    "or by using the --thread-count, --eq-count or --singlethread options:\n"
+	    "* The --thread-count option controls the total number of threads.\n"
+	    "* Increasing the --eq-count option at a fixed --thread-count will reduce the number\n"
+	    "  of fuse threads accordingly. The default value for --eq-count is 1.\n"
+	    "* The --singlethread mode will use one thread for handling fuse requests and a\n"
+	    "  second thread for a single event queue, for a total of two threads.\n"
 	    "\n"
 	    "If dfuse is running in background mode (the default unless launched via mpirun)\n"
 	    "then it will stay in the foreground until the mount is registered with the\n"
 	    "kernel to allow appropriate error reporting.\n"
 	    "\n"
-	    "The -o is to allow use of dfuse via fstab or similar and accepts standard mount\n"
-	    "options.  This will be treated as a comma separated list of key=value pairs and\n"
-	    "dfuse will use pool= and container= keys from this string.\n"
+	    "The -o option can be used to run dfuse via fstab or similar and accepts standard\n"
+	    "mount options.  This will be treated as a comma separated list of key=value pairs,\n"
+	    "and dfuse will use pool= and container= keys from this string.\n"
 	    "\n"
-	    "Caching is on by default with short metadata timeouts and write-back data cache,\n"
-	    "this can be disabled entirely for the mount by the use of command line options.\n"
-	    "Further settings can be set on a per-container basis via the use of container\n"
-	    "attributes.  If the --disable-caching option is given then no caching will be\n"
-	    "performed and the container attributes are not used, if --disable-wb-cache is\n"
-	    "given the data caching for the whole mount is performed in write-back mode and\n"
-	    "the container attributes are still used\n"
+	    "Caching is on by default, with short metadata timeouts and write-back data cache\n"
+	    "enabled. The caching behavior for a dfuse mount can be controlled by command line.\n"
```

Comment:
Extra newline.  My spell checking is complaining about **behavior** but perhaps that's a US/UK thing?
```suggestion
	    "enabled. The caching behavior for a dfuse mount can be controlled by command line\n"
```

Reply:
fixed .